wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -306,7 +306,7 @@
 "conversation.create.group_name.placeholder" = "Group name";
 "conversation.create.guidance.empty" = "At least 1 character";
 "conversation.create.guidance.toolong" = "Too many characters";
-"conversation.create.options.title" = "Conversation options";
+"conversation.create.options.title" = "CONVERSATION OPTIONS";
```

Comment:
I'm not aware of what the convention is regarding naming localizable strings, but maybe should we align on making them all lowercased and handle any formatting where it's needed at the codebase level.

Reply:
We are transitioning from formatting within the codebase to managing all formatting directly in the Localizable.strings file. This change allows us to make adjustments without requiring code modifications, streamlining our localization process and improving overall efficiency.